api/grpcserver/debug_service.go

```diff
@@ -65,3 +69,61 @@ func (d DebugService) Accounts(_ context.Context, in *empty.Empty) (*pb.Accounts
 func (d DebugService) NetworkInfo(ctx context.Context, _ *empty.Empty) (*pb.NetworkInfoResponse, error) {
 	return &pb.NetworkInfoResponse{Id: d.identity.ID().String()}, nil
 }
+
+// ProposalsStream streams all proposals confirmed by hare.
+func (d DebugService) ProposalsStream(_ *emptypb.Empty, stream pb.DebugService_ProposalsStreamServer) error {
```

Comment:
where can i find definition and pb files for DebugService_ProposalsStreamServer?

Reply:
https://github.com/spacemeshos/api/pull/162